packages/frontend/src/components/message/MessageListAndComposer.tsx

```diff
@@ -69,6 +71,14 @@ type Props = {
   accountId: number
 }
 
+function fullPath(file: ParsedPath) {
+  return file.dir + '/' + file.name + file.ext
```

Comment:
This won't work on Windows.

Reply:
Actually, it should work because Rust uses an OS-agnostic Path type, which parses fine in core and works from that point onwards on all OS.